### PR TITLE
Nerfs plasma cutter family.

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl: 
balance: Plasma cutter is now a heavy weapon
balance: Plasma cutter now has negative damage scaling
balance: Plasma cutter can no longer dismember
/:cl:

[why]: # Outclasses the KA family too much. Simply too strong in low pressure, beating the laser gun and offering dismemberment and incredible ammunition and dual wield capability. Needs a nerf.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Things that won't be accepted: Balance changes.

**Comment by Nilons** [comment]
It does like 10 damage and is useless for everything except delimbing, which its not even that good at.

**Comment by PKPenguin321** [comment]
Also, there is a `balance` tag for changelogs you can use instead of `add` and `del`

**Comment by LanCartwright** [comment]
I'm sorry @ChangelingRain Joan.
It's my fault. I've been out of the loop, I recall hearing an antag freeze, but I didn't realise it covered this.

**Comment by Repukan** [comment]
The dream is not dead

**Comment by KorPhaeron** [comment]
Not sure if PvE balance changes should be covered by the freeze, especially if the other half of the equation (the mobs) are going to be touched in the mean time

**Comment by Bawhoppen** [comment]
This is a blatant balance change how can you be indecisive? It's like someone changing esword force = 50

**Comment by KorPhaeron** [comment]
We've been accepting other lavaland balance PRs so a PVE oriented PR is acceptable for now I guess

**Comment by adamkad1** [comment]
laser guns are not supposed to be good in low pressure.  and plasma cutters now have short range, you just cant git gud

**Comment by Expletive** [comment]
@KorPhaeron This isn't a PVE balance change, though. It removes dismemberment from plasma cutters. Dismemberment is primarily effective against player controlled mobs, i.e humans.

File: code/modules/projectiles/projectile/special.dm [hunk 1]
```diff
@@ -233,6 +240,12 @@
 	range = 5
 	mine_range = 5
 
+/obj/item/projectile/plasma/adv/Range()
+	..()
+	damage += -1
```
Comment (Cyberboss) [inline code/modules/projectiles/projectile/special.dm]: The advance plasma cutter loses damage faster than the normal one?

Comment (LanCartwright) [inline code/modules/projectiles/projectile/special.dm]: oh bugger that was a mistake. I don't recall why I did this a week ago.

Comment (Repukan) [inline code/modules/projectiles/projectile/special.dm]: I asked ye to nerf this

**Comment by GunHog** [comment]
First, the dismemberment nerf violates the freeze, as this is only effective in PvP. Secondly, how does this outclass the KA when cutters do a lot less damage (Base KA = 40, Adv cutter = 28). The range/fire rate of the cutter makes it a superior weapon to the base KA, but a properly upgraded KA will be serious competition in combat. 

The Advanced Plasma Cutter should outclass the base KA - it is not a roundstart item, but an upgrade. A miner can at the same time he is getting a plasma cutter get upgrades for his KA which will make it a strong combat option. All it really "needs" is to prevent dual wielding them. Nothing else.

Oh, and your change to the base /plasma type needs to be revised such that it does not affect turret and mecha shots. I would ask you to fix this now to save me the work of reverting your changes to the non-handheld versions later.

**TL;DR:** Nerfing the dual wielding is fine, but it does not need anything else.

**Comment by Expletive** [comment]
@GunHog Has dual wielding been buffed? Last I remember, I wanted it to just stop existing, since it was so inaccurate that it never hit what you were aiming at, and I just wanted to have my accelerator and plasma cutter both in my hands, but not trying to fire them at the same time because it's objectively worse than firing them individually.

**Comment by 4dplanner** [comment]
@expletive: Help intent stops firing both at once

**Comment by Repukan** [comment]
Dual wielding got adjusted so laser weapons don't have such a wide spread.

Eat shit if you want to dual wield them, dual delimbing + high DPS on high range is cancer

**Comment by LanCartwright** [comment]
KA is 1 shot per 1.6
PC is 1 shot per 0.4

**Comment by SgtxRootbeer** [comment]
make this only apply to the hand-held versions, not the ones from the aux base, or for mining mechs.
mechs cannot choose a pka. this was accounted for, and balanced previously. aux base has similar issues.

**Comment by Cyberboss** [comment]
@LanCartwright
